gradle/libs.versions.toml

```diff
@@ -0,0 +1,49 @@
+[versions]
+kotlin = "1.9.21"
+kotlinx-metadata = "0.7.0"
+asm = "9.6"
+slf4j = "1.8.0-alpha2"
+junit = "4.12"
+maven = "3.5.3"
+node-gradle = "3.1.1"
+node = "8.11.1"
+npm = "5.7.1"
+rhino = "1.7.10"
+
+[libraries]
+
+# Kotlin dependencies
+kotlin-gradlePlugin = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
+kotlin-atomicfu = { module = "org.jetbrains.kotlin:atomicfu", version.ref = "kotlin" }
+kotlin-stdlib = { module = "org.jetbrains.kotlin:kotlin-stdlib", version.ref = "kotlin" }
+kotlin-stdlibJdk8 = { group = "org.jetbrains.kotlin", name = "kotlin-stdlib-jdk8", version.ref = "kotlin" }
```

Comment:
Are the Kotlin stdlib dependencies necessary? They should be added automatically by KGP (since it doesn't define `kotlin.stdlib.default.dependency=false`), so they don't need to be added here, and can be removed from the build scripts?

Maybe this PR is just accurately converting the existing dependencies, and the Kotlin stdlib dependencies can be revisited in later PR?